src/lib/components/workflow/dropdown-filter/workflow-datetime-filter.svelte

```diff
@@ -53,9 +57,18 @@
       value = 'All Time';
       timeField = 'StartTime';
     } else {
-      value = custom ? 'Custom' : timeFilter.value;
+      value =
+        custom || !columnOrderedDurations.includes(timeFilter?.value)
```

Comment:
Currently, if a user enters a search or the url has a query that isn't included in `columnOrderedDuration` the value shows up as `undefined` in the dropdown. This change defaults it to `Custom`.